specs/commandline/verify.md

```diff
@@ -54,47 +68,68 @@ An example of `trustpolicy.json`:
+
+### Verify signatures on an OCI artifact stored in a registry
+
+Configure trust store and trust policy properly before using `notation verify` command.
+
+```shell
+
+# Prerequisites: Signatures are stored in a registry referencing the signed OCI artifact
 # Configure trust store by adding a certificate file into trust store named "wabbit-network" of type "ca"
 notation certificate add --type ca --store wabbit-networks wabbit-networks.crt
 
-# Configure trust policy by creating a JSON document named "trustpolicy.json" under directory "{NOTATION_CONFIG}"
-# Example on Linux
-cat <<EOF > $HOME/.config/notation/trustpolicy.json
-{
-    "version": "1.0",
-    "trustPolicies": [
-        {
-            "name": "wabbit-networks-images",   // Name of the policy.
-            "registryScopes": [ "registry.wabbit-networks.io/software/net-monitor" ],          // The registry artifacts to which the policy applies.
-            "signatureVerification": {          // The level of verification - strict, permissive, audit, skip.
-                "level" : "strict" 
-            },
-            "trustStores": [ "ca:wabbit-networks" ], // The trust stores that contains the X.509 trusted roots.
-            "trustedIdentities": [                   // Identities that are trusted to sign the artifact.
-                "x509.subject: C=US, ST=WA, L=Seattle, O=wabbit-networks.io, OU=Finance, CN=SecureBuilder"
-            ]
-        }
-    ]
-}
-EOF
+# Create a JSON file named "trustpolicy.json" under directory "{NOTATION_CONFIG}".
 
-# Verify signatures on a container image 
-notation verify registry.wabbit-networks.io/software/net-monitor:v1
+# Verify signatures on the supplied OCI artifact identified by the digest
+notation verify localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
 ```
 
-### Verify signatures on an OCI artifact stored in a registry (Trust store and trust policy are configured properly)
+An example of output messages for a successful verification:
+
+```text
+Successfully verified for localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+```
+
+### Verify signatures on an OCI artifact identified by a tag
+
+A tag is resolved to a digest first before verification.
 
 ```shell
 # Prerequisites: Signatures are stored in a registry referencing the signed OCI artifact
 
-# Verify signatures on an OCI artifact identified by the digest
-notation verify registry.wabbit-networks.io/software/net-monitor@sha256:abcd1234abcd1234abcd1234abcd1234abcd1234abcd1234abcd1234abcd1234
+# Verify signatures on an OCI artifact identified by the tag
+notation verify localhost:5000/net-monitor:v1
+```
+
+An example of output messages for a successful verification:
+
+```text
+Warning: Always verify artifact using digest(`@sha256:...`) rather than a tag(`:latest`) because tags are mutable and a tag reference can point to a different artifact than the one verified.
```

Comment:
I think we should reword this warning? Because it is okay to verify with a tag and pull the artifact with the digest later on.

Reply:
Check the latest update.